packages/foreman/foreman/foreman.spec

```diff
@@ -554,26 +554,6 @@ Meta package to install requirements for oVirt compute resource support.
 %files ovirt
 %{_datadir}/%{name}/bundler.d/ovirt.rb
 
-%package compute
```

Comment:
Removing this does break the foreman_docker and foreman_one packages. It's also in the `foreman-ovirt` subpackage requirement so that won't pass repoclosure either.